helm-service/controller/deployment_handler.go

```diff
@@ -154,6 +154,27 @@ func (h *DeploymentHandler) catchupGeneratedChartOnboarding(deploymentStrategy k
 	return h.onboarder.OnboardGeneratedChart(userChartManifest, event, deploymentStrategy)
 }
 
+func (h *DeploymentHandler) getPortOfDeployedService(e keptnv2.EventData) (string, error) {
+	userChartManifest, err := h.getHelmExecutor().GetManifest(helm.GetReleaseName(e.Project, e.Stage, e.Service, false),
+		e.Project+"-"+e.Stage)
+
+	if err != nil {
+		return "", err
+	}
+	services := helm.GetServices(userChartManifest)
+	if len(services) > 0 {
+		if len(services[0].Spec.Ports) > 0 {
+			for _, port := range services[0].Spec.Ports {
+				if port.Protocol == corev1.ProtocolTCP {
+					return fmt.Sprintf("%d", port.Port), nil
+				}
+			}
+		}
```

Comment:
This will obviously be problematic for services that expose more than one port. 
Thinking of a service with multiple exposed ports, we could introduce a best practice to always take the lowest port that is exposed, what do you think @grabnerandi @bacherfl ?

Reply:
Agreed, I think taking the lowest port per default is a reasonable approach

Reply:
Okay, could you please adapt the implementation here to follow that approach?

Reply:
Done - the implementation now follows that approach